index.html

```diff
@@ -0,0 +1,1156 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <title>TNW CLONE</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="" />
+    <meta name="author" content="Jorge Cerón & Marcos Hernández" />
+    <link rel="stylesheet" type="text/css" href="css/main.css" />
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <link rel="stylesheet" type="text/css" href="css/responsive.css" />
+    <link rel="icon" type="image/png" href="./img/favicon.png" />
+  </head>
+
+  <body>
+    <header>
+      <a href="#">
+        <span id="orange_container">
+          <span id="logo_mobile_navbar"> </span>
+        </span>
+      </a>
+      <nav>
```

Comment:
The nav bar is exactly the same! Looks great! :ok_hand:
It would look even better if you can add the functionality to show on the top when scrolling through the page as in the original. :monocle_face:
![image](https://user-images.githubusercontent.com/44822765/208265437-e8346bba-ad57-4c69-9714-5a976ffefb64.png)

Reply:
Almost there but please try to check the code, find some errors and bugs and explain why you are requesting those changes. While you are checking the code consider the aspects below.

Lack of [semantic HTML tags use](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
Not applying [HTML & CSS best practices correclty](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
Bugs missed the first time the project was built.